turtle_tf2_py/setup.cfg

```diff
@@ -1,4 +1,4 @@
 [develop]
-script-dir=$base/lib/turtle_tf2_py
+script_dir=$base/lib/turtle_tf2_py
 [install]
-install-scripts=$base/lib/turtle_tf2_py
+install_scripts=$base/lib/turtle_tf2_py
```

Comment:
I am not sure if we need to modify these ones.

Reply:
@kurshakuz yes, for rolling it would be "_", while for foxy, it may need to be "-". So does it have any influences?